calibrators/marker_radar_lidar_calibrator/README.md

```diff
@@ -0,0 +1,168 @@
+# marker_radar_lidar_calibrator
+
+A tutorial for this calibrator can be found [here](../docs/tutorials/marker_radar_lidar_calibrator.md)
+
+## Purpose
+
+The package `marker_radar_lidar_calibrator` allows extrinsic calibration between radar and 3d lidar sensors used in autonomous driving and robotics.
+
+Currently, the calibrator only supports radars whose detection interface includes distance and azimuth angle, but do not offer elevation angle. For example, ARS 408 radar can be calibrated with this tool. Also, note that the 3d lidar should have a resolution that is high enough to scan several points on the [radar reflector](#radar-reflector) (calibration target).
+
+## Inner-workings / Algorithms
+
+The calibrator is designed to estimate the transformation between radar and lidar sensors. It starts by pinpointing the central points of reflectors within lidar pointclouds and radar objects, then aligns these points for precise matching. An SVD-based and a yaw-only rotation estimation algorithm are applied to these matched points to estimate the transformation.
```

Comment:
Bad use of the comma on the second sentence

Reply:
Couldn't get this one

Reply:
Replace the comma by an and

Also the `central points of reflectors`  is not what I think you want to say.

Reply:
Thanks, fixed in https://github.com/tier4/CalibrationTools/pull/164/commits/cbd750746caef8746d49ef3976fa6421c467319e

Reply:
Sadly, it is sill incorrect.
The way it is written means that that the center of the reflector appears as a point in the point cloud (even more by the choice of words of pinpointing).

Reply:
True, fixed in https://github.com/tier4/CalibrationTools/pull/164/commits/54d81743c6127c0e78c51cbdc9c449f2cee2eefc

Reply:
Still wrong, but sure